test/balancer_test.go

```diff
@@ -718,8 +718,6 @@ func (s) TestEmptyAddrs(t *testing.T) {
 
 	// Remove all addresses.
 	pfr.UpdateState(resolver.State{})
-	// Wait for a ResolveNow call on the pick first client's resolver.
```

Comment:
Oh, I remember why I had to delete this. Remember before how flow was update grpc.ClientConn with something, and then within resolver.ClientConn it would call poll()...which would then call ResolveNow. This now with my this new behavior change would make it block forever. Let me go ahead and delete callback.